extensions/ql-vscode/src/databases/local-databases-ui.ts

```diff
@@ -359,6 +363,12 @@ export class DatabaseUI extends DisposableObject {
     );
   }
 
+  private async handleChooseDatabaseFoldersParentFromPalette(): Promise<void> {
+    return withProgress(async (progress) => {
+      await this.chooseDatabasesParentFolder(progress);
+    });
+  }
```

Comment:
Updated according to your suggestions with minor refine.

I also removed the title of this progress bar, because otherwise the whole message will be too long and doesn't display the actual step message of each database.

Currently will be:
![image](https://github.com/user-attachments/assets/dae42815-85d0-46ae-92d8-542c77f995b6)

Before:
![image](https://github.com/user-attachments/assets/be625914-7ed0-4e2a-96e5-954211965092)